### DISCUSSION TITLE
https://blog.3place.cn/3place/fsd/FSD040.html

### DISCUSSION THREAD

**Discussion by giscus** [hbaolong/hbaolong.github.io#42] [General]
# https://blog.3place.cn/3place/fsd/FSD040.html

https://blog.3place.cn/3place/fsd/FSD040.html

**Comment by picklecai** [comment]
战争、贸易和海盗行为
是三位一体，不可分开。

gede门儿清得很，半买半抢。

**Comment by picklecai** [comment]
富贵而感到缺少，
乃是我们最大的烦恼。

季氏富于周公，而求也为之聚敛而附益之。

**Comment by picklecai** [comment]
拿伯的葡萄园，贾赦的扇子
